spaceR/spaceR/Utilities/Constants.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Constants.swift
+//  spaceR
+//
+//  Created by Kieran Woodrow on 2022/02/17.
+//
+
+import Foundation
+
+struct Constants{
+    //Endpoint URl for getting ALL rockets
+    static let getAllRocketsUrl = URL(string: "https://api.spacexdata.com/v4/rockets")
```

Comment:
Using guard might not seem much different to using if, but with guard your intention is clearer: execution should not continue if your conditions are not met. I would suggest adding guard instead of if as if just check if the condition is true and the url may not exist so then we would prefer to stop all code execution

For example guard let getAllRocketsUrl = URL(string: "https://api.spacexdata.com/v4/rockets") else {
return
}

Reply:
Thanks Kirsty